client/test/utils/pseudorandom.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 
 
-let seed = 0;
+let seed = process.env.PSEUDORANDOMSEED === undefined ? Math.random() :
+    parseInt(process.env.PSEUDORANDOMSEED, 10);
+
+// eslint-disable-next-line no-console
+console.log('SEED is', seed);
```

Comment:
No, but it should allow us to debug when they happen. Ie "tests fail when SEED IS ...."